docs/applications/simple-bank-account/README.md

```diff
@@ -207,16 +207,15 @@ try (ClientService clientService = injector.getInstance(ClientService.class)) {
 
 ```java
 JsonObject argument = Json.createObjectBuilder().add("id", "010-123456789").build();
-ContractExecutionResponse response = clientService.executeContract("create-account", argument);
+ContractExecutionResult response = clientService.executeContract("create-account", argument);
 ```
 
 will execute the `CreateAccount` contract with argument `{"id": "010-123456789"}`, as we did above. Note that we call the contract using the supplied id `create-account` that we chose when registering the contract.
 
-The result of executing the contract is a `ContractExecutionResponse`. It contains, among other things, a status code, error message, and result, each of which can be obtained respectively as
+The result of executing the contract is a `ContractExecutionResult`. It contains, proofs and result, each of which can be obtained respectively as
 
 ```java
-response.getStatus();
-response.getMessage();
+response.getProofs()
 response.getResult();
```

Comment:
```suggestion
result.getResult();
```

Reply:
Thank you for review comments.
Changes updated in 07087f3cf6c0ba8d8349ec1e9d18e9d8cc8f0a61